scripts/seed.py

```diff
@@ -0,0 +1,162 @@
+# Command to run : python manage.py shell --settings=settings.dev  < scripts/seed.py
+
+from django.core.files.uploadedfile import SimpleUploadedFile
+from django.contrib.auth.models import User
+from django.utils import timezone
+
+from allauth.account.models import EmailAddress
+
+from datetime import timedelta
+from faker import Factory
+import random
+
+from challenges.models import Challenge, ChallengePhase, DatasetSplit, Leaderboard, ChallengePhaseSplit
+from hosts.models import ChallengeHostTeam, ChallengeHost
+from participants.models import Participant, ParticipantTeam
+
+
+def create_admin_user():
+    username = "admin"
+    email = "admin@gmail.com"
+    User.objects.create_user(
+        email=email,
+        username=username,
+        password="password",
+        is_staff=True,
+        is_superuser=True,
+    )
+    print "Super user made with Username: admin and Password: password"
+
+
+def create_user():
+    fake = Factory.create()
+    username = fake.user_name()
+    email = "%s@gmail.com" % (username)
+    user = User.objects.create_user(
+        email=email,
+        username=username,
+        password="password",
+    )
+    EmailAddress.objects.create(user=user, email=email, verified=True, primary=True)
+    return user
+
+
+def create_challenge_host_team(user):
+    fake = Factory.create()
+    team_name = "%s %ss" % (fake.city(), fake.color_name())
+    team = ChallengeHostTeam.objects.create(
+        team_name=team_name,
+        created_by=user,
+    )
+    print "Challenge Host Team created"
+    return team
+
+
+def create_challenge_host(user, team):
+    ChallengeHost.objects.create(user=user, team_name=team, status="Self", permissions="Admin")
+    print "Challenge Host created"
+
+
+def create_challenge(creator_team):
+    fake = Factory.create()
+    created_challenge = Challenge.objects.create(
+        title="%s Challenge" % (fake.first_name()),
+        description=fake.paragraph(),
+        terms_and_conditions=fake.paragraph(),
+        submission_guidelines=fake.paragraph(),
+        evaluation_details=fake.paragraph(),
+        evaluation_script=SimpleUploadedFile(fake.file_name(extension="txt"),
+                                             'Dummy file content', content_type='text/plain'),
+        creator=creator_team,
+        published=True,
+        enable_forum=True,
+        anonymous_leaderboard=False,
+        start_date=timezone.now() - timedelta(days=2),
+        end_date=timezone.now() + timedelta(days=1),
```

Comment:
Also, can you create one challenge as the `Future Challenge` and one as the `Past Challenge`?